src/ApiService/ApiService/onefuzzlib/AutoScale.cs

```diff
@@ -215,7 +215,7 @@ public AutoScaleOperations(ILogTracer log, IOnefuzzContext context)
 
 
     public AutoscaleProfile DefaultAutoScaleProfile(string queueUri, long scaleSetSize) {
-        return CreateAutoScaleProfile(queueUri, 1L, scaleSetSize, scaleSetSize, 1, 10.0, 1, 5.0);
+        return CreateAutoScaleProfile(queueUri, Math.Min(scaleSetSize, 1L), scaleSetSize, scaleSetSize, 1, 10.0, 1, 5.0);
```

Comment:
Math.Max?

Reply:
brain fart on my part. 

this is not needed at all - bug was in other part of the code

Reply:
min would be correct, since it's ether 1 or 0 when scaleset size is 0 (but that was different bug)